### PR TITLE
Add per erasure set info in server info

### PR DESCRIPTION
**Pull request by vadmeste**
Add per erasure set information, that will be shown in mc admin info 
output or AdminClient.ServerInfo().

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: info-commands.go [hunk 1]
```diff
@@ -236,6 +247,8 @@ type InfoMessage struct {
 	Services     Services           `json:"services,omitempty"`
 	Backend      interface{}        `json:"backend,omitempty"`
 	Servers      []ServerProperties `json:"servers,omitempty"`
+
+	Pools map[int]map[int]ErasureSetInfo `json:"pools,omitempty"`
```
Comment (klauspost) [inline info-commands.go:251]: `map[int]` cannot be used in JSON. It must be `map[string]` or arrays.

Comment (harshavardhana) [reply info-commands.go:251]: doesn't go automatically convert this to string? @klauspost

**Review by klauspost — CHANGES_REQUESTED** [review]

**Comment by vadmeste** [comment]
@klauspost it works fine for me. You just need to test https://github.com/minio/minio/pull/16583

**Comment by vadmeste** [comment]
Oh, also you need to edit go.mod in github.com/minio/mc

```
replace github.com/minio/madmin-go/v2 => github.com/vadmeste/madmin-go/v2 v2.0.7-0.20230209161037-4883f168c2c6
```

then `mc admin info --json <alias> | jq .info.pools`

File: info-commands.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (klauspost) [reply info-commands.go:251]: Oh wow, they added that in Go 1.7

> In earlier versions of Go, this package only supported encoding and decoding maps using keys with string types. Go 1.7 adds support for maps using keys with integer types: the encoding uses a quoted decimal representation as the JSON key.

Seems risky to use such a recent feature :D

**Review by klauspost — APPROVED** [review]
LGTM

**Pull request merged**